src/test/java/resourceTests/PairResourceTests/PairResourceFindPairsBetweenThreePersonsTest.java

```diff
@@ -46,37 +47,37 @@ public void setUp() {
         expectedFirstPersonSlotsTimes = MockDataUtil.createSlotTimeList(1, 3);
         expectedSecondPersonSlotsTimes = MockDataUtil.createSlotTimeList(1, 5);
 
-        Person firstPerson = MockDataUtil.createPerson((long) 1, "FIRST", isDev, isTest, isOps);
+        Person firstPerson = MockDataUtil.createPerson((long) 1, "FIRST", isDev, isTest, isOps, isOther);
```

Comment:
It probably would be cause I didn't like that I had to add isOther in like ~30 other places. But I think this might be candidate for issue, cause there is already waiting pull request for fixing some pairs issues and it might cause more conflicts.